docs/installation.md

```diff
@@ -125,7 +125,16 @@ __Install SDK Wheel__
 !!! note
     Replace &lt;SDK WHEEL LINK&gt; with the SDK link you copied earlier from the console.
 
-__Info about xailient sdk installed__
+__Start the Xailient Daemon which activates your license__
+
+```bash
+(env) $ python3 -m xailient.install
```

Comment:
Should this be `sudo ./xailient-install` instead? Or have we changed the way to activate license recently?

Reply:
Good catch. I'm so used to using docker container where sudo isn't needed that I wasn't thinking of the real case of installing this onto a device. I'll fix this now.